.github/workflows/ci.yml

```diff
@@ -311,6 +312,25 @@ jobs:
             securitypolicy.exe
             uvmboot.exe
             zapdir.exe
+            rootfs.exe
+
+  build_linux:
+    needs: [test-windows, test-linux, integration-tests]
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - run: go build ./internal/tools/rootfs
+
+      - uses: actions/upload-artifact@v2
+        if: ${{ github.event_name == 'pull_request' }}
+        with:
+          name: linux_binaries
+          path: |
+            rootfs
```

Comment:
Is there anything about the tool that would prohibit it from building on Windows? I'd prefer to just build it on the windows machines if so